NumericOutcome.md

```diff
@@ -0,0 +1,561 @@
+    * [Optimizations](#optimizations)
+  * [Algorithms](#algorithms)
+* [General Payout Curves](#general-payout-curves)
+  * [Design](#design)
+  * [Curve Serialization](#curve-serialization)
+  * [General Function Evaluation](#general-function-evaluation)
+  * [Optimized Evaluation During CET Calculation](#optimized-evaluation-during-cet-calculation)
+  * [Rounding Intervals](#rounding-intervals)
+    * [Rounding Interval Serialization](#rounding-interval-serialization)
+* [Contract Execution Transaction Calculation](#contract-execution-transaction-calculation)
+* [Contract Execution Transaction Signature Validation](#contract-execution-transaction-signature-validation)
+* [Authors](#authors)
+
+## Adaptor Points with Multiple Signatures
+
+Given public key `P` and nonces `R1, ..., Rn` we can compute `n` individual signature points for
+a given event `(d1, ..., dn)` in the usual way: `si * G = Ri + H(P, Ri, di)*P`.
+To compute a composite adaptor point for all events which agree on the first `m` digits, where
+`m` is any positive number less than or equal to `n`, the sum of the corresponding signature
+points is used: `s(1..m) * G = (s1 + s2 + ... + sm) * G = s1 * G + s2 * G + ... + sm * G`.
+
+When the oracle broadcasts its `n` signatures `s1, ..., sn`, the corresponding adaptor secret can be
+computed as `s(1..m) = s1 + s2 + ... + sm` which can be used to broadcast the CET.
+
+#### Rationale
+
+This design allows implementations to re-use all [transaction construction code](Transactions.md) without modification
+because every CET needs as input exactly one adaptor point just like in the single-nonce setting.
+
+Another design that was considered was adding keys to the funding output so that parties could collaboratively
+construct `m` adaptor signatures and where `n` signatures are put on-chain in every CET which would reveal
+all oracle signatures to both parties when a CET is published.
+This design's major drawbacks is that it creates a very distinct fingerprint and makes CET fees significantly worse.
+Additionally it leads to extra complexity in contract construction.
+This design's only benefit is that it results in simpler (although larger) fraud proofs.
+
+The large multi-signature design was abandoned because the above proposal is sufficient to generate fraud proofs.
+If an oracle incorrectly signs for an event, then only the sum of the digit signatures `s(1..m)`
+is recoverable on-chain using the adaptor signature which was given to one's counter-party.
+This sum is sufficient information to determine what was signed however as one can iterate through
+all possible composite adaptor points until they find one whose pre-image is the signature sum found on-chain.
+This will determine what digits `(d1, ..., dm)` were signed and these values along with the oracle
+announcement and `s(1..m)` is sufficient information to generate a fraud proof in the multi-nonce setting. 
+
+## Contract Execution Transaction Compression
```

Comment:
It might be useful to add a motivation section here? 

basically why do we need tx compression? Why can't we just do the dumb thing and have a CET for every payout.